config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -60,13 +60,14 @@ periodics:
ould.be.consumable.from.pods.in.volume.with.defaultMode.set.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].ConfigMap.should.be.consumable.from.pods.in.volume.with.mappings.and.Item.mode.set.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].ConfigMap.should.be.consumable.from.pods.in.volume.with.mappings.as.non-root.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Downward.API.volume.should.set.DefaultMode.on.files.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Downward.API.volume.should.set.mode.on.item.file.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].EmptyDir.volumes.*\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].HostPath.should.give.a.volume.the.correct.mode.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Projected.configMap.should.be.consumable.from.pods.in.volume.as.non-root.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Projected.configMap.should.be.consumable.from.pods.in.volume.with.defaultMode.set.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Projected.configMap.should.be.consumable.from.pods.in.volume.with.mappings.and.Item.mode.set.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Projected.configMap.should.be.consumable.from.pods.in.volume.with.mappings.as.non-root.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Projected.secret.should.be.consumable.from.pods.in.volume.as.non-root.with.defaultMode.and.fsGroup.set.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Projected.secret.should.be.consumable.from.pods.in.volume.with.defaultMode.set.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Projected.secret.should.be.consumable.from.pods.in.volume.with.mappings.and.Item.Mode.set.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Secrets.should.be.consumable.from.pods.in.volume.as.non-root.with.defaultMode.and.fsGroup.set.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Secrets.should.be.consumable.from.pods.in.volume.with.defaultMode.set.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Secrets.should.be.consumable.from.pods.in.volume.with.mappings.and.Item.Mode.set.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Subpath.Atomic.writer.volumes.should.support.subpaths.with.configmap.pod.\\[Conformance\\]|\\[sig-storage\\].Subpath.Atomic.writer.volumes.should.support.subpaths.with.configmap.pod.with.mountPath.of.existing.file.\\[Conformance\\]|\\[sig-storage\\].Subpath.Atomic.writer.volumes.should.support.subpaths.with.downward.pod.\\[Conformance\\]|\\[sig-storage\\].Subpath.Atomic.writer.volumes.should.support.subpaths.with.projected.pod.\\[Conformance\\]|\\[sig-storage\\].Subpath.Atomic.writer.volumes.should.support.subpaths.with.secret.pod.\\[Conformance\\]|\\[sig-storage\\].Projected.downwardAPI.should.set.DefaultMode.on.files.\\[NodeConformance\\].\\[Conformance\\]|\\[sig-storage\\].Projected.downwardAPI.should.set.mode.on.item.file.\\[NodeConformance\\].\\[Conformance\\]"
```

Comment:
this is a scary line. bug AFAIK ginkgo does accept config.

Reply:
I don't understand what you are referring to? The huge skip regex? The rationale behind that is that not all conformance tests can be run on Windows and there is no mechanism of skipping.

However: this patch just got merged: https://github.com/kubernetes/kubernetes/pull/73204 thus I will update the PR to skip LinuxOnly tag

Reply:
> I don't understand what you are referring to

just wanted to comment that ginkgo does not support reading the regex from config and this line is huge.

Reply:
Gotcha! To the best of my knowledge there isn't any possibility to send it from config. Anyways, LinuxOnly label solved it :)